### DISCUSSION TITLE
A question about Integrate Paddle backend in devel

### DISCUSSION THREAD

**Discussion by HydrogenSulfate** [deepmodeling/deepmd-kit#4101] [General]
Hi @njzjz , we found multiple backend is supported in `devel` branch, so can we now add paddle backend to that branch(including core code and unitests)? I have verified paddle can be easily added, but I have no idea if `devel` now is a stable branch as we can develop based on it.

**Comment by njzjz** [comment]
The multiple-backend framework in the `devel` branch has been stable. Some other issues need to be resolved before the stable version but are unrelated to the multiple-backend framework.
